### PR TITLE
* Changes for computing chebyshev distance.

### PR DESCRIPTION
**Pull request by MishraShivendra**
These changes would introduce a few functions to compute chebyshev distance between two data points. Please take a look.
-Shivendra

### CODE REVIEW

**Comment by montanaflynn** [comment]
Thanks for the contribution! 

The only feedback I have is to use camelCase instead of snake_case for the variable names. https://golang.org/doc/effective_go.html#mixed-caps

Once those changes are done I'll merge this :)

**Comment by montanaflynn** [comment]
I found a few other minor stylistic and things, I left comments to the code here in GitHub but here is the final output:

```go
func ComputeChebyshevDistance(dataPointX, dataPointY []float64) (distance float64, err error) {

	if len(dataPointX) == 0 || len(dataPointY) == 0 {
		return math.NaN(), EmptyInput
	}

	if len(dataPointX) != len(dataPointY) {
		return math.NaN(), SizeErr
	}

  var tempDistance  float64

	for i = 0; i < len(dataPointY); i++ {
		tempDistance = math.Abs(dataPointX[i] - dataPointY[i])
		if distance < tempDistance {
			distance = tempDistance
		}
	}

	return distance, nil
}
```

File: chebyshev_distance.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// Contact : shivendra_mishra@live.com
+//
+package stats
+
+import (
+	"math"
+)
+
+// Compute Chebyshev distance between two data points 
+func ComputeChebyshevDistance( data_point_x, data_point_y []float64 ) ( distance float64,
```
Comment (montanaflynn) [inline chebyshev_distance.go]: The spaces are not idiomatic or consistent with the rest of the codebase. `go fmt` will fix these :)

File: chebyshev_distance.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import (
+	"math"
+)
+
+// Compute Chebyshev distance between two data points 
+func ComputeChebyshevDistance( data_point_x, data_point_y []float64 ) ( distance float64, 
+                                                                        err error ) {
+	var (
+		temp_distance float64
```
Comment (montanaflynn) [inline chebyshev_distance.go]: As mentioned in https://golang.org/doc/effective_go.html#mixed-caps it's preferred to use camelCase for variables and function names (or any names really)

File: chebyshev_distance.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		i             int
```
Comment (montanaflynn) [inline chebyshev_distance.go]: This is defined later in the loop `for i = 0; i < len(data_point_y); i++ `

**Comment by MishraShivendra** [comment]
Hi Montana,
Thanks for those inputs. Pushed the suggested changes.
-Shivendra

**Comment by montanaflynn** [comment]
Thanks Shivendra! I'll merge this in now.

The documentation is auto-generated and available here: 

https://godoc.org/github.com/montanaflynn/stats

**Pull request merged**